src/entities/prediction_request.py

```diff
@@ -4,17 +4,15 @@
 
 class PredictionRequest:
 
-    def __init__(self, dataset, model, rawModel, additionalInfo=None, doaMatrix=None):
+    def __init__(self, dataset, model, doaMatrix=None):
         self.dataset = dataset
         self.model = model
-        self.rawModel = rawModel
-        self.additionalInfo = additionalInfo
+        self.rawModel = model.rawModel
+        self.additionalInfo = model.additionalInfo
         self.doaMatrix = doaMatrix
 
 
 class PredictionRequestPydantic(BaseModel):
     dataset: Any
     model: Any
-    additionalInfo: Any
     doaMatrix: Any = None
```

Comment:
Minor change: instead of doaMatrix this should be renamed to doa, since it will cover more methods and not only leverage